activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Fix a regression with division by a Duration object
+
+    Before:
+        rental_end = DateTime.parse('01-01-2017 12:45:00').utc.in_time_zone('Europe/Berlin')
+        rental_start = DateTime.parse('01-01-2017 12:00:00').utc.in_time_zone('Europe/Berlin')
+        (rental_end - rental_start) / 1.hour
+        => 2700.0 hours
+    
+    After:
+        rental_end = DateTime.parse('01-01-2017 12:45:00').utc.in_time_zone('Europe/Berlin')
+        rental_start = DateTime.parse('01-01-2017 12:00:00').utc.in_time_zone('Europe/Berlin')
+        (rental_end - rental_start) / 1.hour
+        => 0.75
```

Comment:
@koic  I think the behavior was broken in Rails 5.1.1, too. If I divide a time duration by another time duration the units cancel out.